docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -0,0 +1,211 @@
+[[sentinel]]
+= Sentinel
+
+_The Defender Sentinel service allows you to monitor transactions for Addresses by specfying Events, Functions, and Transaction conditions and notifying via email, slack, and autotasks (more to come)._
+
+[[use-cases]]
+== Use cases
+
+Use a Sentinel whenever you need to know about or respond to transactions involving your smart contract.  Your Sentinel will watch every transaction send the transactions you care about to your notification methods of choice.
+
+* *Monitor* your *sensitive functions* like transferOwnership, pause
+* *Alert* when transactions are failing against your smart contract
+* *Respond* to transactions by executing *Autotask* logic when key events happen 
+* *Integrate* with your *existing tools* through slack, email or custom *Autotask* integration
+* *Know* when an unexpected volume of transactions occur
+
+[[whats-in-an-sentinel]]
+== What's in an Sentinel?
+
+A Sentinel watches transactions for a given address, filters those transactions by any conditions that you specify, then notifies you when those transactions occur.
+
+IMPORTANT: At this time, Sentinels do not monitor internal function calls.
+
+[[specify-address]]
+=== Specifying an Address
```

Comment:
This feels out of context coming from the "what's in a sentinel" section. As an intro, I'd add a `== Creating a sentinel` section, explain that "creating a sentinel involves specifying the address to monitor, defining matching rules, and choosing the notification channels", and only then go into the creation process.